README.md

```diff
@@ -24,6 +24,7 @@ This client supports the following Google Cloud Platform services at a [Beta](#v
 * [BigQuery](#bigquery-beta) (Beta)
 * [Cloud Pub/Sub](#cloud-pubsub-beta) (Beta)
 * [Cloud Spanner API](#cloud-spanner-api-beta) (Beta)
+* [Cloud Video Intelligence API](#cloud-video-intelligence-api-alpha) (Beta)
```

Comment:
The anchor needs to be changed to: `#cloud-video-intelligence-api-beta`

Reply:
Done.